message_tagging_service/tagging_service.py

```diff
@@ -349,10 +352,10 @@ def tag_build(nvr, dest_tags, koji_session):
                 task_id = 1
             else:
                 task_id = koji_session.tagBuild(tag, nvr)
```

Comment:
Hm, I think you have to wait for the result of task returned by `tagBuild` and see if it finished successfully to know that the build has been really tagged.

Reply:
Your current code will catch some cases when tagBuild can fail early in the Koji, but if it fails later in the tagBuild task itself, the code will not detect it.

Reply:
That's right. It's [intentionally](https://github.com/fedora-modularity/message-tagging-service/pull/33) done this way.

Reply:
@hanzz That was discussed before in the PR Luiz lineked. As a result, for current stage, we decided to not wait for the task and just continue once tagBuild call finishes and include task id in the message. And that is also the reason we changed the topic name from `mts.build.tagged` to `mts.build.tag.requested`.